backend/routes/postsRoutes.js

```diff
@@ -0,0 +1,50 @@
+import { Router } from 'express'
+import { db } from '../lib/db'
+import { posts_db_name } from '../Utilities/API_utilities'
+
+
+const postsRouter = Router()
+
+postsRouter.get('/api/posts', (req, res) => {
+    db.getAll(posts_db_name).then(result => {
+        console.log('all posts', result, 'posts_db_name', posts_db_name)
+        return res.json(result)
+    }).catch(error => {
+        console.log('error getting all libraries', error)
+        return res.json(error)
```

Comment:
Probably want to add a failed HTTP status code here, potentially like:
```js
return res.status(500).json(error)
```
This will make it easier for you to immediately see failed requests in your JavaScript console.

Reply:
+1